extras/ams_wrapper/src/api/models/vehicle_detection_adas_model.py

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+import sys
+import json
+from logger import get_logger
+from api.models.model import Model
+
+logger = get_logger(__name__)
+
+
+class VehicleDetectionAdas(Model):
+
+    def load_default_labels(self):
+        labels_path = os.path.abspath(__file__).replace(".py",".json")
+        try:                                                                          
+            with open(labels_path, 'r') as labels_file:                               
+                data = json.load(labels_file)                                         
+                self.labels = data
+        except Exception as e:                                                        
+            logger.exception("Error occurred while opening labels file: {}".format(e))
+            sys.exit(1)                                                               
+        return                                                                   
+
+
+    def postprocess_inference_output(self, inference_output: dict) -> str:
+        from api.types import Tag, Rectangle, SingleEntity, Entity
+
+        result_array = inference_output["detection_out"]
```

Comment:
We could consider adding output name to the model configuration, like we do with input:
https://github.com/openvinotoolkit/model_server/blob/ams/extras/ams_wrapper/src/config.py#L23

Reply:
Done.

Reply:
Is this dictionary somewhere validated? Is it possible that the key does not exist?

Reply:
Result is validated in functional test:
boxes_count = str(json_response).count("box")

print("detected boxes:" + str(boxes_count))
assert boxes_count == 2

try:
    format_check = json.loads(json_response)
except Exception as e:
    print("json loads exception:" + str(e))
    assert False

print("format_check:" + str(format_check))
assert format_check["subtype"] == "vehicle-detection"